configuration/exodus-lambda-deploy.yaml

```diff
@@ -36,7 +55,7 @@ Resources:
         Name: !Sub ${project}-cache-policy-${env}
         ParametersInCacheKeyAndForwardedToOrigin:
           CookiesConfig:
-            CookieBehavior: none
+            CookieBehavior: all
```

Comment:
Can we please use a whitelist rather than "all" ?   Setting "all" may make the cache less effective as responses will not be cacheable across, e.g., two different browsers which are passing some random cookies unrelated to exodus.

Reply:
Yeah, I'm just not sure what should be listed. From what I can tell, our cookies don't have names. Or it looking for domain names? Documentation doesn't specify, unfortunately, what's sought.

Reply:
The cookie names are those starting with "CloudFront-", e.g. "CloudFront-Key-Pair-Id", from cookies_for_policy method.

Reply:
Ah, okay. Thanks!